### ISSUE TITLE
[iOS / Critical] Tap gesture recognizer does not work

### ISSUE THREAD

**Issue by adrianknight89** [sthewissen/Xamarin.Forms.PancakeView#36]
Adding a tap gesture to `PancakeView` does not work on iOS. When a view is tapped, the event handler is not triggered. See the repro. If you run it on Android, it will work.

[App1545.zip](https://github.com/sthewissen/Xamarin.Forms.PancakeView/files/3513356/App1545.zip)

**Comment by sthewissen** [comment]
Oddly enough, when adding a child it does work. I'm guessing somehow the touch surface isn't sized accordingly when no childs are present. I'd have to look into it further.

```
<pancakeview:PancakeView BackgroundColor="Green" WidthRequest="250" HeightRequest="50">
   <pancakeview:PancakeView.GestureRecognizers>
       <TapGestureRecognizer Tapped="TapGestureRecognizer_Tapped" />
    </pancakeview:PancakeView.GestureRecognizers>
    <Label Text="test" />
</pancakeview:PancakeView>

**Comment by adrianknight89** [comment]
My actual code has a child which is a Label, and it didn't work in my case, but if I remember correctly, I set `InputTransparent` to true on the child so the label doesn't/shouldn't capture the tap. So when you release the fix, can you also test this scenario?

**Comment by bcaceiro** [comment]
I have noticed the same in iOS. Even with a child element (image)

**Comment by bcaceiro** [comment]
Any news on this?

**Comment by sthewissen** [comment]
No. I have not had any time to figure it out yet. Suggestions are welcome.

**Comment by sthewissen** [comment]
Apparently, because I set a new NativeView in the iOS renderer I also need to take the GestureRecognizers and apply them again, because at that point they're already added and applied. Should be fixed in 1.1.8-beta.
